src/engine/profiler.cc

```diff
@@ -56,14 +55,13 @@ Profiler::Profiler()
 #endif
 
   this->profile_stat = new DevStat[cpu_num_ + gpu_num_ + 1];
```

Comment:
Should this be deleted in the destructor?

Reply:
That's for a different PR.  I actually tried to once, but Profiler is currently a static object right now and things get ugly when it shuts down and tries to delete this.  I will probably be refactoring some of this eventually.